rosdep/python.yaml

```diff
@@ -219,6 +219,10 @@ opcua-pip:
   ubuntu:
     pip:
       packages: [opcua]
+open3d-pip:
```

Comment:
For both of these, our style is `python-open3d-pip` (so we can also add a python3 key as needed).  Also, since this is pip, we may as well add it for Fedora and debian as well.

Reply:
done